src/ucs/arch/cpu.h

```diff
@@ -18,6 +18,21 @@ typedef enum ucs_cpu_model {
     UCS_CPU_MODEL_LAST
 } ucs_cpu_model_t;
 
+/* CPU flags */
+typedef enum ucs_cpu_flag {
+    UCS_CPU_FLAG_UNKNOWN    = (-1),
```

Comment:
maybe we should use #define then? i would like to give room for more possible bits (consider this will also run on ppc and arm machines)